underfs/swift/src/main/java/alluxio/underfs/swift/SwiftUnderFileSystem.java

```diff
@@ -261,7 +273,92 @@ public boolean mkdirs(String path, boolean createParent) throws IOException {
 
   @Override
   public boolean mkdirs(String path, MkdirsOptions options) throws IOException {
-    return true;
+    if (path == null) {
+      return false;
```

Comment:
I was going to add a similar comment, but then I checked that all of our other object storage UFS do the same. It should be addressed in a separate PR.